skymp5-functions-lib/index.ts

```diff
@@ -164,6 +164,164 @@ export const placeholder = (mp: Mp, self: PapyrusObject | null, args: PapyrusVal
   return undefined;
 }
 
+// TODO: Move these types to a better place
+type SPExchangeRet = {
+  formId: number, returnNumber: number,
+}
+
+type SPExchangeReq = {
+  formId: number, requiredCount: number,
+  returnSelect: 'random' | 'none', returns: SPExchangeRet[],
+}
+
+type SPExchange = {
+  formId: number, commissionItem: number,
+  commissionSize: number, startMessage: number,
+  failMessage: number, finishMessage: number,
+  reqs: SPExchangeReq[],
+}
+
+// TODO: Get these values from external config file
+const spExchanges: SPExchange[] = [
+  {
+    formId: 0x064d2b03,
+    commissionItem: 0x0f,
+    commissionSize: 1,
+    startMessage: 0x0635d1f0,
+    failMessage: 0x0635d1f2,
+    finishMessage: 0x0635d1f1,
+    reqs: [
+      {
+        formId: 0x0f,
+        requiredCount: 30,
+        returnSelect: 'random',
+        returns: [
+          {
+            formId: 0x0300353a,
+            returnNumber: 1,
+          }, {
```

Comment:
Looks a bit strange, cuz all JSON pretty-printers (and you a few lines below) do like so
```suggestion
          },
          {
```

Reply:
Copypaste went wrong